run.py

```diff
@@ -30,10 +47,18 @@ def main():
     lang = settings.value('Language')
     locale_folder = op.join(BASE_PATH, 'locale')
     install_gettext_trans_under_qt(locale_folder, lang)
+    # Handle OS signals
+    setUpSignals()
+    # Let the Python interpreter run every 500ms to handle signals.
+    from PyQt5.QtCore import QTimer
+    timer = QTimer()
```

Comment:
Could we please have a comment that mentions the Qt's runloop then? The role of this piece of code is hard to grasp without the context of the PR.